mlflow/utils/autologging_utils/safety.py

```diff
@@ -191,9 +191,7 @@ def with_managed_run(autologging_integration, patch_function, tags=None):
     """
 
     def create_managed_run():
-        managed_run = mlflow.start_run()
-        if tags:
-            try_mlflow_log(mlflow.set_tags, tags)
```

Comment:
I forgot why we need to do this.

Reply:
One difference I rememeber is if we run create a run this way, the `managed_run` object doesn't contain `tags`.

Reply:
I think we previously needed to do this because TensorFlow autologging checked for the `mlflow.autologging` tag on the active run to determine whether or not an autologging session was in progress, which is no longer necessary (removed here: https://github.com/mlflow/mlflow/commit/4e68f960d4520ade6b64a28c297816f622adc83e#diff-0ea13712d8aad26f69240bff7787b862ab88122c9c9e1936ac4af414efd58227L1063).